x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/home/index.tsx

```diff
@@ -61,7 +61,9 @@ export function EPMHomePage() {
 
 function InstalledPackages() {
   useBreadcrumbs('integrations_installed');
-  const { data: allPackages, isLoading: isLoadingPackages } = useGetPackages();
+  const { data: allPackages, isLoading: isLoadingPackages } = useGetPackages({
+    experimental: true,
```

Comment:
this param will be needed when we switch to having a toggle/option to turn on experimental packages (#64869) instead of showing by default because we should always show all installed packages regardless of release type. I'd like to keep this param here as it would be easy to overlook the necessary behavior here when we remove `experimental: true` from `useGetPackages`